src/components/explore/AggregationsOverTimeGraph.svelte

```diff
@@ -260,20 +240,16 @@ $: if (referenceTextElement && referenceBackgroundElement) {
 
  <g slot=annotation let:top let:bottom let:left let:xScale let:yScale>
   {#if hovered.datum}
-      <line 
-      x1={xScale(hovered.datum.label)}
-      x2={xScale(hovered.datum.label)}
-      y1={top}
-      y2={bottom}
-      stroke=var(--cool-gray-200)
-      stroke-dasharray=2,1
-    />
+    
+    <TrackingLine x={hovered.datum.label} />
+    <TrackingLabel x={hovered.datum.label} align=top background=white label=Hov. />
+
     {#each plotValues(hovered.datum.label, hovered.datum[yAccessor], metricKeys, xScale, yScale) as {x, y, bin}, i (bin)}
         <Springable value={[x, y]} let:springValue>
           <circle 
             cx={x}
             cy={y}
-            r=2
+            r=3
```

Comment:
In the long run we should avoid magic numbers; it honestly doesn't matter for now we have them in quite a few places.

Reply:
agreed, these should probably be in `constants.js`